groonga-apt-source/debian/rules

```diff
@@ -5,16 +5,20 @@
 #export DH_VERBOSE=1
 # This has to be exported to make some magic below work.
 export DH_OPTIONS
+export GROONGA_HOME := $(shell mktemp -d)
```

Comment:
Since it is not Groonga's home, wouldn't it be better to use different variable name?